apps/charterafrica/src/components/FeaturedPostCard/FeaturedPostCard.js

```diff
@@ -96,24 +97,33 @@ const FeaturedPostCard = React.forwardRef(function FeaturedPostCard(
                 day: "numeric",
               })}
             </Typography>
-
-            <LineClampedRichTypography
+            {/* Line height is constant across all screens at 24 px * 6 lines. Has to be wrapped in another div to overcome */}
```

Comment:
We don't need this since `FeaturedPost` only appears in desktop so it should go without saying.